TUTORIAL.md

```diff
@@ -452,7 +452,7 @@ import Gallery from './gallery.component';
         
 // ... previously written code
 
-                {captures.length > 0 && <Gallery captures={captures}/>}
+                {this.state.captures.length > 0 && <Gallery captures={this.state.captures}/>}
```

Comment:
Sorry, captures is already being destructured from state before the renderer. https://github.com/foysalit/rn-tutorial-vedo/blob/master/src/camera.page.js#L48